apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -25,7 +25,8 @@ const TutorialExplorer = React.createClass({
     locale: React.PropTypes.string.isRequired,
     backButton: React.PropTypes.bool,
     roboticsButton: React.PropTypes.bool,
-    showSortBy: React.PropTypes.bool.isRequired
+    showSortBy: React.PropTypes.bool.isRequired,
+    disableTutorials: React.PropTypes.arrayOf(React.PropTypes.string).isRequired
```

Comment:
nit: i might called this `disabledTutorials` instead, to make it a little more clear that this is a list rather than a function